cmd/flow-collector/handlers.go

```diff
@@ -166,3 +168,45 @@ func (c *Controller) collectorHandler(w http.ResponseWriter, r *http.Request) {
 		fmt.Fprintf(w, "%s", *response.Body)
 	}
 }
+
+func (c *Controller) promqueryHandler(w http.ResponseWriter, r *http.Request) {
+	client := http.Client{}
+
+	urlOut := c.FlowCollector.Collector.PrometheusUrl + "query?" + r.URL.RawQuery
+	proxyReq, err := http.NewRequest(r.Method, urlOut, nil)
```

Comment:
Don't you need to pass the r.Body and r.Header to the proxyReq?
Without it, POST data won't be passed through or am I missing something?

Reply:
Presently, the console does not POST to the query api